cypress/fixtures/docs/validations.md

```diff
@@ -421,6 +422,86 @@ The 'schemaOnly` validation type allows to skip content validation and stops at
 
 ```
```

Comment:
FCS supports screenshot validation. This validation is used to validate the screenshot captured by the platform. The screenshot validation object should be added to the validation objects. 
this doesn't make sense

Reply:
addressed the comment